torch/utils/data/sampler.py

```diff
@@ -59,10 +59,10 @@ class SequentialSampler(Sampler[int]):
     """
     data_source: Sized
 
-    def __init__(self, data_source):
+    def __init__(self, data_source: Sized) -> None:
```

Comment:
Can you add a comment of `type: ignore` at the end to pass the mypy error?
It is caused by `Dataset` that is not guaranteed with `__len__` function.

Reply:
Sure thing, I was wondering what caused this issue in the syntax checker

Reply:
I am so sorry for the wrong instruction. The mypy error doesn't come from this line (You can remove the type: ignore)
The comment of `type: ignore` should be placed at the following line: https://github.com/pytorch/pytorch/blob/c61778355c3b237a25be92f80b74fd87e83d76e3/torch/utils/data/dataloader.py#L269
Since we add type hint for the argument of SequentialSampler as `Sized`, the dataset in the dataloader is not guaranteed with __len__.

Reply:
No worries!